.github/workflows/test.yml

```diff
@@ -27,19 +27,19 @@ jobs:
       matrix:
         # maximize CI coverage of different platforms and python versions while minimizing the
         # total number of jobs. We run all pytest splits with the oldest supported python
-        # version (currently 3.9) on windows (seems most likely to surface errors) and with
+        # version (currently 3.10) on windows (seems most likely to surface errors) and with
         # newest version (currently 3.12) on ubuntu (to get complete coverage on unix).
         config:
           - os: windows-latest
-            python: "3.9"
+            python: "3.10"
             resolution: highest
             extras: ci,optional
           - os: ubuntu-latest
-            python: ">3.9"
+            python: ">3.10"
             resolution: lowest-direct
             extras: ci,optional
           - os: macos-latest
-            python: "3.10"
```

Comment:
Need confirm: A bit unsure why we pinned 3.10 for MacOS, to test some version in between highest and lowest?

Reply:
> perhaps to test some version in between highest and lowest?

that's right!

Reply:
Thanks for confirming